madmom/features/tempo.py

```diff
@@ -847,11 +851,13 @@ def __init__(self, min_bpm=MIN_BPM, max_bpm=MAX_BPM, act_smooth=ACT_SMOOTH,
             hist_smooth=hist_smooth, fps=fps, online=online, **kwargs)
         # save additional variables
         if self.online:
-            self.buffer = BufferProcessor(int(buffer_size * self.fps))
+            self._buffer = BufferProcessor(int(buffer_size * self.fps))
```

Comment:
I'd prefer having access to the buffer, thus the leading underscore is not necessary.